src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/jpa/entity/repository/UnallocatedCaseEntity.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.jpa.entity.repository
+
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType.IDENTITY
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "unallocated_cases")
+data class UnallocatedCaseEntity(
```

Comment:
I think we should add validation if we know what are mandatory fields and have NotNull? For example val initial_appointment: LocalDateTime? = null, should be annotated @NotBlank possibly.